README.md

```diff
@@ -0,0 +1,5 @@
+# Watershed (Under Construction :construction: :construction_worker: :sweat_drops:)
+
+Water is part of something bigger than a drop (us and the tech industry) and also cool since Carleton does have a lot of water surrounding it I think it fits well. And the fact it's made of triangle shapes also shows how it's built from various pieces, sorta like a hack.
```

Comment:
I think this needs to be a more formal definition, and not just a copy/paste of Andrew's message.